geotrek/core/tests/test_commands.py

```diff
@@ -229,3 +230,353 @@ def test_load_paths_within_spatial_extent_no_srid_geom(self):
         value = Path.objects.first()
         self.assertEqual(value.name, 'lulu')
         self.assertEqual(value.structure, self.structure)
+
+
+@skipIf(not settings.TREKKING_TOPOLOGY_ENABLED, 'Test with dynamic segmentation only')
+class ReorderTopologiesPathAggregationTest(TestCase):
+    def setUp(self):
+        """
+        ⠳               ⠞
+          ⠳           ⠞
+            ⠳       ⠞
+              ⠳   ⠞
+                ⠿
+              ⠞   ⠳
+            ⠞       ⠳
+        1 ⠞           ⠳ 2
+        ⠞               ⠳
+        """
+        self.path_1 = PathFactory.create(geom=LineString(Point(700000, 6600000), Point(700100, 6600100),
+                                                         srid=settings.SRID))
+        self.path_2 = PathFactory.create(geom=LineString(Point(700000, 6600100), Point(700100, 6600000),
+                                                         srid=settings.SRID))
+        self.path_1_a = Path.objects.get(geom=LineString(Point(700000, 6600000), Point(700050, 6600050),
+                                                         srid=settings.SRID))
+        self.path_1_b = Path.objects.get(geom=LineString(Point(700050, 6600050), Point(700100, 6600100),
+                                                         srid=settings.SRID))
+        self.path_2_a = Path.objects.get(geom=LineString(Point(700000, 6600100), Point(700050, 6600050),
+                                                         srid=settings.SRID))
+        self.path_2_b = Path.objects.get(geom=LineString(Point(700050, 6600050), Point(700100, 6600000),
+                                                         srid=settings.SRID))
+
+    def get_geometries(self):
+        geometries = []
+        for pathagg in PathAggregation.objects.all():
```

Comment:
For consistency I would filter the queryset with the topology's ID in the same way you do in the tests.